client/dashboard/domains/domain-overview/featured-card-site.tsx

```diff
@@ -0,0 +1,22 @@
+import { __ } from '@wordpress/i18n';
+import { Domain } from '../../data/domain';
+import { Site } from '../../data/site';
+import OverviewCard from '../../sites/overview-card';
+import SiteIcon from '../../sites/site-icon';
+
+interface Props {
+	domain: Domain;
+	site: Site;
+}
+
+export default function FeaturedCardSite( { domain, site }: Props ) {
+	return (
+		<OverviewCard
+			title={ __( 'Site' ) }
+			heading={ site.name }
+			link={ site.URL }
```

Comment:
This should be an external link no?


```suggestion
			externalLink={ site.URL }
```

Reply:
I see the mockup in DOTDASH-315 uses a chevron icon instead of an external link icon 🤔 

Maybe we should have it link to `/sites/$siteSlug`?

Reply:
> Maybe we should have it link to /sites/$siteSlug?

It's probably true. I changed it to be `/sites/site.slug` 👌